skyline-vscode/react-ui/src/utils/parsers.js

```diff
@@ -0,0 +1,97 @@
+import Ajv from "ajv";
+import {
+  gpuPropertyList,
+  deploymentScatterGraphColorSize,
+} from "../data/properties";
+import { cloudProviderSchema } from "../schema/CloudProvidersSchema";
+
+const ajv = new Ajv({ allErrors: true }); // to report all validation errors (rather than failing on the first errors)
+const validate = ajv.compile(cloudProviderSchema);
+
+class ResponseBuffer {
```

Comment:
Rename this so it is more clear what is used for

Reply:
class was deleted